doc/whats_new/v1.5.rst

```diff
@@ -70,6 +70,10 @@ See :ref:`array_api` for more details.
   :pr:`27904` by :user:`Eric Lindgren <elindgren>`, `Franck Charras <fcharras>`,
   `Olivier Grisel <ogrisel>` and `Tim Head <betatim>`.
 
+- :func:`sklearn.metrics.mean_tweedie_deviance` now supports Array API compatible
+  inputs.
+  :pr:`28106` by :user:`Thomas Li <lithomas1>`
```

Comment:
Since we cut 1.5.X earlier this week, this entry will have to be pushed to `v1.6.rst` (after a merge of `main` into this branch).